plenum/server/node.py

```diff
@@ -2132,7 +2125,12 @@ def sendInstanceChange(self, view_no: int,
                                 self.monitor.prettymetrics))
             msg = self._create_instance_change_msg(view_no, suspicion.code)
             self.send(msg)
-            self._record_inst_change_msg(msg, self.name)
```

Comment:
Why did we remove this method?

Reply:
updated, replaced it with other one having more appropriate name